scripts/stubsabot.py

```diff
@@ -435,21 +438,22 @@ async def determine_action(stub_path: Path, session: aiohttp.ClientSession) -> U
 
     diff_info = await get_diff_info(session, stub_info, pypi_info, relevant_version)
     if diff_info is not None:
-        github_repo_path, old_tag, new_tag, diff_url = diff_info
+        *_, diff_url = diff_info
         links["Diff"] = diff_url
```

Comment:
I assume this is more readable than:
```suggestion
        links["Diff"] = diff_info[3]
```

Reply:
I just realized you can do `*_, links["Diff"] = diff_info`. The worst of both worlds 😃 (looks neat though)